### DISCUSSION TITLE
Welcome to ariadne_django Discussions!

### DISCUSSION THREAD

**Discussion by maxmorlocke** [reset-button/ariadne_django#21] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
 
Rules are simple: Be kind and assume positive intent.  Fail at either of those and say goodbye.

**Comment by lorddaedra** [comment]
@maxmorlocke

Hello! Thank you for creation integration with Ariadne. 

I know, you are looking for maintainer. May be this org may help.
https://jazzband.co/about/guidelines

**Reply by rafalp** [reply]
+1 here, getting ariadne_django under Jazzband seems to be the way.
